pyproject.toml

```
@@ -38,11 +38,12 @@ fasteners = "^0.17.3"
 flask = "^2.1"
 flask-cors = "^3.0.7"
 flask-executor = "^0.10"
+flask-login = "=0.6.1" # https://github.com/meltano/meltano/issues/6548
```

Is this dep syntax the same as

```suggestion
flask-login = "==0.6.1" # https://github.com/meltano/meltano/issues/6548
```

?

---

[The Poetry docs](https://python-poetry.org/docs/dependency-specification/#exact-requirements) don't have any info about using `=` or `==`. Apparently to specify an exact requirement, you're supposed to use the version number on its own. Seems like `=`, `==`, and the plain version number are all equivalent.

---

Yeah, Poetry does its own dep parsing, though [PEP 440](https://peps.python.org/pep-0440/#version-matching) prefers `==`. Regardless, I don't have any strong preference